packages/smooth_app/lib/themes/theme_manager.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class UserThemePreference {
+  // ignore: non_constant_identifier_names
```

Comment:
Please don't ignore the recommendations.
Write `static const String THEME_STATUS = 'THEMESTATUS';` instead